os-autoinst-openvswitch

```diff
@@ -22,6 +22,10 @@ sub init_switch {
     $self->{BRIDGE} = $ENV{OS_AUTOINST_USE_BRIDGE};
     $self->{BRIDGE} //= 'br0';
 
+    ## coolo and richard terrible hack to make bridge
+    system("/usr/bin/ovs-vsctl add-br $self->{BRIDGE}");
+    system("/usr/sbin/ifup $self->{BRIDGE}");
+
```

Comment:
The bridge configuration is persistent, so I'd expect that this fails on second reboot. Or is the bridge deleted somewhere?